CMakeLists.txt

```diff
@@ -39,6 +39,19 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
   FIND_PACKAGE(Threads)
 endif(CMAKE_SYSTEM_NAME MATCHES "Linux")
 
+if(WIN32)
```

Comment:
in the commit message, it reads

> * update FindBoost to accept clang as an acceptable option on Windows

but the FindBoost change is missing in this commit. i think we agreed to move it to a separated commit in this changeset.

Reply:
Oh, right. That's a leftover, I forgot to update the commit message. Sorry about that.